app/javascript/modCenter/singleArticle/singleArticle.test.jsx

```diff
@@ -29,15 +47,32 @@ describe('<SingleArticle />', () => {
       />,
     );
 
-  it('should have no a11y  violations', async () => {
+  const renderSingleArticle2 = (article = testArticle2) =>
+    render(
+      <SingleArticle
+        id={article.id}
+        title={article.title}
+        path={article.path}
+        publishedAt={article.publishedAt}
+        cachedTagList={article.cachedTagList}
+        user={article.user}
+      />,
+    );
```

Comment:
You can reuse the `renderSingleArticle` here and pass in `testArticle2` :) The `(article = testArticle2)` here is a default argument.

Reply:
ahhh yes! I don't my brain was firing on all cylinders at this point 😅